DiceKeys/Views/DiceKey/DieView.swift

```diff
@@ -172,15 +178,19 @@ struct DiceKeyFaceArray: View {
 }
 
 struct DieView: View {
-    let face: Face
+    var face: Face?
+    var partialFace: PartialFace?
     let dieSize: CGFloat
     let linearFractionOfFaceRenderedToDieSize: CGFloat = CGFloat(5)/8
     var penColor: Color = Color.black
     var faceSurfaceColor: Color = Color.white
     var faceBorderColor: Color?
+    
+    var computedPartialFace: PartialFace { partialFace ?? PartialFace(face!) }
```

Comment:
Is it guaranteed face will not be nil when partialFace is nil? It will crash here otherwise

Reply:
Either face or partialFace should be set.

I think the proper way to require either A or B to be set is to write two initializers.